### PR TITLE
ign -> gz Header Migration : sdformat

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/711

### CODE REVIEW

File: test/integration/CMakeLists.txt [hunk 1]
```diff
@@ -58,7 +58,7 @@ set(tests
   world_dom.cc
 )
 
-if (PYTHONINTERP_FOUND AND PY_PSUTIL)
+if (Python3_Interpreter_FOUND AND PY_PSUTIL)
```
Comment (chapulina) [inline test/integration/CMakeLists.txt:61]: This seems out of place. I see that [92d4e9c](https://github.com/ignitionrobotics/sdformat/pull/1008/commits/92d4e9c26d98d1403fb64c3c7c875e4cab50365e) has been included in this PR, but that PR should already be on `main` as [8a3e5a8](https://github.com/ignitionrobotics/sdformat/commit/8a3e5a828241405516e6601d353ed7b0993c1b8b)🤔

File: include/sdf/system_util.hh [hunk 2]
```diff
@@ -27,7 +27,7 @@
 /** \def SDFORMAT_HIDDEN
  * Use to represent "symbol hidden" if supported
  */
-#define SDFORMAT_HIDDEN IGNITION_SDFORMAT_HIDDEN
+#define SDFORMAT_HIDDEN GZ_SDFORMAT_HIDDEN
```
Comment (chapulina) [inline include/sdf/system_util.hh:30]: Oops, I think we're not ready to change this.

**Review by chapulina — COMMENTED** [review]
Just 2 lines to double-check. The rest of the code and CI look good.

File: include/sdf/system_util.hh (hunk 2 above)
Comment (scpeters) [reply include/sdf/system_util.hh:30]: agreed

Comment (methylDragon) [reply include/sdf/system_util.hh]: Interesting that the script edited one but not the other :thinking: Maybe I manually discarded one of them and forgot to discard the other. Oops.

https://github.com/ignitionrobotics/sdformat/pull/1008/commits/b751698896d4c9d87e09d6cd09d8dcff5e54b5ad

File: test/integration/CMakeLists.txt (hunk 1 above)
Comment (methylDragon) [reply test/integration/CMakeLists.txt]: Oops

https://github.com/ignitionrobotics/sdformat/pull/1008/commits/b751698896d4c9d87e09d6cd09d8dcff5e54b5ad

**Comment by scpeters** [comment]
since this is just one commit, it can be squash-merged

**Review by scpeters — APPROVED** [review]

**Review by chapulina — APPROVED** [review]

**Pull request merged**